pages/discharge-upgrade-instructions/index.md

```diff
@@ -1,12 +0,0 @@
----
-layout: page-react-sidebar.html
-title: How To Apply For A Discharge Upgrade
-heading: How to apply for a discharge upgrade
-entryname: discharge-upgrade-instructions
-display_title: Discharge upgrade
-description: Want to apply for a military discharge upgrade or correction? Get step-by-step instructions depending on your specific situation. If your discharge is upgraded, you'll be eligible for VA benefits you earned while serving.
-keywords: upgrade discharge, military discharge upgrade, discharge upgrade
-order: 6
-spoke: Get records
-collection: records
----
```

Comment:
Migrated here, https://github.com/department-of-veterans-affairs/vets-website/pull/10660/files#diff-ac6f3dc1df5ee7b05b337198c5214ab2R5. Breadcrumbs confirmed to look okay.